### ISSUE TITLE
Formulate a recommendation for the use of skos labels and definitions for ABox data

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#780]
`skos:definition` is well-suited to class, property, and taxonomic term definitions - I.e., terms that we create and define and can give whatever label and definition we choose. It is _not_ well-suited to instance data. For example, a person has a description ("brown hair, blue eyes, 6' tall, mechanical engineer, married with 3 children") but not a definition.

There are some cases where a definition is appropriate: e.g., we can _define_ what the US Supreme Court is; e.g., "The Supreme Court of the United States is the highest court in the federal judiciary of the United States. It has ultimate appellate jurisdiction over all U.S. federal court cases, and over state court cases that involve a point of federal law." We might also provide a description, which would add non-defining characteristics such as the current court having 9 members and meeting at 1 First St NE, Washington, DC 20543.

I would like to say the same applies to `skos:prefLabel`, where `gist:name` is generally more appropriate for, say, persons, than a label. People don't have labels. However, other considerations in this case are:

* Applications may be dependent on prefLabels for every resource.
* ABox resources like persons may have more than one name, among which one is the preferred name. We could either use `skos:prefLabel` for this purpose, or define a property `gist:preferredName`. On the other hand, preferred name may be context- or application-specific. For example, Susan might prefer to be referred to by her nickname, "Susie." Or a particular software system might prefer to list her as "Jones, Susan M." while another prefers "Susan M. Jones." This problem arises with `skos:prefLabel` as well, of course. 

My proposal is as follows:
* Recommend the use of `gist:description` rather than `skos:definition` for most ABox data, with exceptions noted as above for entities that can truly be said to have a real world definition.
* Recommend the use of both `skos:prefLabel` and `gist:name` for ABox data. It's reasonable to assign a label for computational purposes, as long as we understand this has no real world significance, and a name should  be used for the latter instead. If there are multiple names, choose one for `skos:prefLabel`; otherwise the name and label should be the same.  Recommend to users that they add other annotations for preferences in specific contexts (e.g., `:hrName`).

**Comment by uscholdm** [comment]
I agree with the proposal

**Comment by dylan-sa** [comment]
I agree that it is intuitive to think of those features of the Supreme Court as being _definitional_ of it in a sense. They are the features that make it the thing it is, so to speak. In principle, though, I think this kind of definition could be developed for any individual. For example, to be the number 2 is to be the successor of 1. I know some have even tried to make the case that a person's definition/essence would have to do with their particular origin, parents, etc. Maybe then we could say that `skos:definition` should be reserved for cases where folks have found it useful to carefully enumerate the definitional features (as in the Supreme Court case). (Maybe it is most likely to be used w/ a controlled vocabulary in a data namespace?)

I think the `skos:prefLabel`/`gist:name` recommendation is good too.

**Comment by uscholdm** [comment]
> I agree that it is intuitive to think of those features of the Supreme Court as being definitional of it in a sense.

While I agree with this, note that a definition is also a description,.   The question is whether this justifies using `skos:definition` for the very small number of instances that are not in the CBox. I don't think we lose much by failing to add the fact that in some cases the description is also a definition.  There are two disadvantages.
1. A decision will need to be made for each of possibly 100s of millions of instances
2. When querying to learn about an instance, you will need to check for two different predicates rather than just one.

  Thus, I recommend we stick with using gist:description and not gist:definition for ABox instances.

**Comment by rjyounes** [comment]
To @dylan-sa's point: it seems to me the only _definition_ of a person we might conceivably give is their DNA sequence; I doubt if any combination of the attributes you describe could produce uniqueness. Then again, that could be changed and they would still be the same person. I think a definition of a person or similar object is ruled out.

I'm OK with @uscholdm's proposal to exclude `skos:definition` on ABox instances across the board.

**Comment by uscholdm** [comment]
@rjyounes, @dylan-sa 
SUMMARY PROPOSAL:
- only use `skos:definition` for classes, properties and taxonomic terms.
- use `gist:description` for data instances 
- use `gist:name` only when the individual has a name in the real world. It will often be the same as the `skos:prefLabel` which is necessary for UI purposes.

**Comment by rjyounes** [comment]
@uschold I agree 100%. This just needs to be documented.

**Comment by rjyounes** [comment]
* Use `skos:definition` for classes, properties, taxonomic terms, controlled vocabularies of concepts (e.g., units of measure)
* Use `gist:description` for data instances that are not concepts (e.g., persons, places)
* Use `gist:name` only when the individual has a name in the real world. It will often be the same as the `skos:prefLabel` which is necessary for UI purposes.

**Comment by stevenchalem** [comment]
Closing as duplicate of #427
